plugins/system/shortcut/fields/keyselect.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @package     Joomla.Plugins
+ * @subpackage  System.shortcut
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+
+
+
+class JFormFieldKeyselect extends JFormField
```

Comment:
JFormField import ?

Reply:
please Import the correct namespace and your field should not be J prefixed like all other fieldnames